Traceback (most recent call last):
  File "main.py", line 342, in <module>
    ins = Instructor(args)
  File "main.py", line 37, in __init__
    self.prepare()
  File "main.py", line 87, in prepare
    os.mkdir(args.save_path)
FileNotFoundError: [Errno 2] No such file or directory: './checkpoint/MGND/RELATION'
Traceback (most recent call last):
  File "main.py", line 342, in <module>
    ins = Instructor(args)
  File "main.py", line 37, in __init__
    self.prepare()
  File "main.py", line 87, in prepare
    os.mkdir(args.save_path)
FileNotFoundError: [Errno 2] No such file or directory: './checkpoint/MGND/ENTITY'
Traceback (most recent call last):
  File "main.py", line 342, in <module>
    ins = Instructor(args)
  File "main.py", line 37, in __init__
    self.prepare()
  File "main.py", line 87, in prepare
    os.mkdir(args.save_path)
FileNotFoundError: [Errno 2] No such file or directory: './checkpoint/MGND/HYBRID'
Traceback (most recent call last):
  File "main.py", line 342, in <module>
    ins = Instructor(args)
  File "main.py", line 37, in __init__
    self.prepare()
  File "main.py", line 87, in prepare
    os.mkdir(args.save_path)
FileNotFoundError: [Errno 2] No such file or directory: './checkpoint/MGND/FACT'
